virttest/utils_test/qemu/__init__.py

```diff
@@ -186,6 +186,34 @@ def clear_win_driver_verifier(driver, vm, timeout=300):
         session.close()
 
 
+@error_context.context_aware
+def check_win_driver(test, driver, vm, timeout=300):
+    """
+    Check if driver is installed and state is running for windows guest.
+    :param test: QEMU test object.
+    :param driver: The driver which needs enable the verifier.
+    :param vm: VM object.
+    :param timeout: Timeout in seconds.
+    :return: A tuple (status, output) where status is the driver check result
+    and output is the detail information.
+    """
+    session = vm.wait_for_login(timeout=timeout)
```

Comment:
Suggest using an existed `session` instead of creating a new one.

Reply:
ok, pass session parameter.